.rubocop.yml

```diff
@@ -29,6 +29,9 @@ AllCops:
     - 'vendor/**/*'
     - 'engines/*/bin/*'
 
+Layout/EmptyLineBetweenDefs:
+  AllowAdjacentOneLineDefs: true
+
```

Comment:
I feel like it would be better to avoid overriding rules unless we absolutely have to. Any individual rule always seems fine but I wouldn't want to end up back where we were with the long list of special cases.

Reply:
The config `AllowAdjacentOneLineDefs` is on by default in rubocop 1.23 and later - not sure why it's not enabled here as it's a weird one for standard-rb to switch.